src/features/auth/hooks/assign-user-role.ts

```diff
@@ -14,7 +15,7 @@ hooks.addPureHook('POST', 'resin', 'user', {
 				req: permissions.root,
 			},
 			id: {
-				name: 'default-user',
+				name: getNewUserRoleName() || 'default-user',
```

Comment:
It's impossible for `getNewUserRoleName()` to return a null-ish value, so:

```suggestion
				name: getNewUserRoleName(),
```